src/orders/model/RefundAdjustment.ts

```diff
@@ -0,0 +1,6 @@
+export default interface RefundAdjustment {
```

Comment:
Just call this Adjustment.ts and rename the model to Adjusmtent. No need to specify that it's for a refund now. If we do add support for these on other parts of the order we should be able to reuse the same object